platform_umbrella/apps/common_core/lib/common_core/defaults/images.ex

```diff
@@ -363,4 +365,17 @@ defmodule CommonCore.Defaults.Images do
     ver = batteries_included_version()
     "ghcr.io/batteries-included/home-base:#{ver}"
   end
+
+  def __after_compile__(_env, _bytecode) do
+    path = Path.relative_to_cwd("priv/images.json")
```

Comment:
This may not be the best (or even a good?) place or name. Happy to change if there are any better ideas.

Reply:
Do we ever need this again? Seems like a one-shot thing. I would takethe nice json that you generated, load the file into a list via `IncludeResource`. For now we can hand edit the JSON.

Reply:
Are you thinking that we'd maintain the image "registry" in the JSON file and load it into Elixir as opposed to generating JSON from Elixir?